build.gradle

```diff
@@ -46,8 +46,7 @@ buildscript {
         testExtJunit = '1.1.0'
         uiAutomatorVersion = '2.2.0'
         viewPagerVersion = '1.0.0-alpha06'
-        workVersion = '2.1.0-beta01'
-        workVersion = '2.1.0-alpha02'
+        workVersion = '2.1.0'
```

Comment:
I think this has been changed on the `master` .

Reply:
Thanks for the heads up.  We'll make a cleanup pass when we're ready to merge these branches.